### PR TITLE
Easier local development of p-s

### PR DESCRIPTION
**Pull request by TomasTomecek**
* make debugging of p-s easier
* document how to run p-s httpd locally
* put httpd config for p-s in this repo

### CODE REVIEW

File: packit_service/service/app.py [hunk 1]
```diff
@@ -46,8 +47,12 @@ def get_flask_application():
     app.register_blueprint(builds_blueprint)
     s = ServiceConfig.get_service_config()
     # https://flask.palletsprojects.com/en/1.1.x/config/#SERVER_NAME
+    # also needs to contain port if it's not 443
     app.config["SERVER_NAME"] = s.server_name
     app.config["PREFERRED_URL_SCHEME"] = "https"
+    if getenv("DEPLOYMENT") in ("dev", "stg"):
+        app.config["DEBUG"] = True
+    app.logger.setLevel(logging.DEBUG)
```
Comment (jpopelka) [inline packit_service/service/app.py]:
```suggestion
        app.logger.setLevel(logging.DEBUG)
```

Comment (TomasTomecek) [reply packit_service/service/app.py]: I'd set it to debug no matter what - I actually wasn't getting any log messages from flask/werkzeug to be honest

File: files/packit-httpd.conf [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# MIT License
```
Comment (csomh) [inline files/packit-httpd.conf]: Why have this here if it's already in [deployment](https://github.com/packit-service/deployment/blob/master/openshift/cmap-packit-httpd-conf.yml.j2)?

Comment (TomasTomecek) [reply files/packit-httpd.conf]: b/c you literally can't run p-s locally without this file, so I will remove it from the deployment repo

hence better local development

Comment (csomh) [reply files/packit-httpd.conf]: Removing this from the deployment repo will force us to rebuild the service image whenever we would like to tweak httpd-packit.conf in production. Is this what we want?

Comment (csomh) [reply files/packit-httpd.conf]: I assume this change was required to have httpd-packit.conf available when running a local deployment with docker-compose. Local deployments done on ab `oc cluster up` local cluster, using the deployment playbooks already had this file mounted.

**Comment by TomasTomecek** [comment]
recheck

**Pull request merged**